JsonExcel.vue

```diff
@@ -205,9 +211,12 @@ export default {
       //Data
       data.map(function(item) {
         for (let key in item) {
-          let escapedCSV = '=\"' + item[key] + '\"'; // cast Numbers to string
-          if (escapedCSV.match(/[,"\n]/)) {
-            escapedCSV = '"' + escapedCSV.replace(/\"/g, '""') + '"';
+          let escapedCSV = item[key];
+          if (_self.escapeCsv) {
+            escapedCSV = '=\"' + escapedCSV + '\"'; // cast Numbers to string
+            if (escapedCSV.match(/[,"\n]/)) {
+              escapedCSV = '"' + escapedCSV.replace(/\"/g, '""') + '"';
+            }
```

Comment:
accept changes from @jny986 adding a prop to validate if it's necessary to escaped csv values